doc/doc/basics/extractors.md

```diff
@@ -17,10 +17,15 @@ class contains different extractor *styles*:
 
 - Row-wise extractors:
   - [`json_extractor`](#json_extractor): highly flexible and compatible with
-  previous systems, but with limited performance 
+  previous systems, but with limited performance
```

Comment:
Start adopting Phabricator today, and it'll become the contributor's responsibility (as opposed to the maintainer's) to handle conflicts (rebase etc.)! :)